azure-pipelines/dotnet.yml

```diff
@@ -43,6 +43,10 @@ steps:
     testRunTitle: netcoreapp2.2-$(Agent.JobName)
     workingDirectory: src
 
+- powershell: mono $(NUGET_PACKAGES)/xunit.runner.console/2.4.1/tools/net472/xunit.console.exe bin/StreamJsonRpc.Tests/$(BuildConfiguration)/net472/StreamJsonRpc.Tests.dll -vsts -notrait "TestCategory=FailsInCloudTest" -noclass PerfTests
```

Comment:
`-noclass PerfTests` skips tests that crash the test runner on mono due to https://github.com/mono/mono/issues/17994